src/v2/view-builder/views/webauthn/EnrollWebauthnView.js

```diff
@@ -60,14 +60,18 @@ const Body = BaseForm.extend({
     const relatesToObject = this.options.currentViewState.relatesTo;
     const activationData = relatesToObject?.value.contextualData.activationData;
     if (webauthn.isNewApiAvailable()) {
+      const excludeCredentials = activationData.authenticatorSelection?.requireResidentKey === true ?
+        [] :
+        getExcludeCredentials(this.options.appState.get('authenticatorEnrollments').value);
```

Comment:
Can we do an optional chain on value? 
`?.value`

Reply:
this enrollment flow is after use authenticated, it appears that `this.options.appState.get('authenticatorEnrollments')` will not be null